src/appium.sh

```diff
@@ -23,8 +23,15 @@ function prepare_geny_cloud() {
 	    device=$(get_value '.device')
 	    port=$(get_value '.port')
 
-	    echo "Starting \"$device\" with template name \"$template\"..."
-	    instance_uuid=$(gmsaas instances start "${template}" "${device}")
+		if [[ $device != null ]]; then
+			echo "Starting \"$device\" with template name \"$template\"..."
+			instance_uuid=$(gmsaas instances start "${template}" "${device}")
+		else
+			echo "Starting Device with Random name..."
+			random_device_name=$(python3  -c 'import uuid; print(str(uuid.uuid4()).upper())')
```

Comment:
I think you dont need to use python just to create random uuid, you can create random uuid within bash e.g. using uuidgen

Reply:
actually , i have tried that, it's not working on cli, uuidgen is not installed by default
i also tried to install it by `apt-get -qqy install uuid-runtime` and it works, should i add it inside dockerfile?

Reply:
Nah, I will approve it then. Thanks again for the PR